### PR TITLE
Fix broken link in docs

### PR DESCRIPTION
**Pull request by fiunchinho**
**What type of PR is this?**

/kind bug
/kind documentation

**What this PR does / why we need it**:
There is a broken link in the docs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
The root cause is CAPI [renaming `clusterctl config cluster` to `clusterctl generate cluster`](https://github.com/kubernetes-sigs/cluster-api/pull/4778).

**TODOs**:

- [X] squashed commits
- [X] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
Fix broken link in docs after clusterctl changes
```

### CODE REVIEW

File: docs/book/src/topics/machinepools.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ which provides the cloud provider specific resource for orchestrating a group of
...
 for that.
 
 Make sure to set up your Azure environment as described [here](https://cluster-api.sigs.k8s.io/user/quick-start.html).
 
 ```shell
-clusterctl config cluster my-cluster --kubernetes-version v1.19.7 --flavor machinepool > my-cluster.yaml
+clusterctl generate cluster my-cluster --kubernetes-version v1.19.7 --flavor machinepool > my-cluster.yaml
 ```
 
 The template used for this [flavor](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors)
```
Comment (mboersma) [inline docs/book/src/topics/machinepools.md:51]: This link needs to be updated too.

**Comment by alexeldeib** [comment]
well, our link check works 😃

File: docs/book/src/topics/machinepools.md (hunk 1 above)
Comment (fiunchinho) [reply docs/book/src/topics/machinepools.md]: Fixed

**Comment by nader-ziada** [comment]
@fiunchinho can you please add a release note, the PR looks good otherwise

**Comment by fiunchinho** [comment]
Done

**Comment by nader-ziada** [comment]
the links check in the `markdown-link-check` all passed 

/lgtm
/approve

**Pull request merged**